jobbergate-api/tests/apps/job_submissions/test_routers.py

```diff
@@ -350,6 +350,88 @@ async def test_create_job_submission_without_client_id(
     assert response.status_code == status.HTTP_400_BAD_REQUEST
 
 
+@pytest.mark.parametrize("permission", (Permissions.ADMIN, Permissions.JOB_SUBMISSIONS_CREATE))
+async def test_clone_job_submission__success(
+    permission,
+    fill_job_script_data,
+    fill_job_submission_data,
+    client,
+    inject_security_header,
+    tester_email,
+    job_script_data_as_string,
+    synth_session,
+    synth_services,
+):
+    base_job_script = await synth_services.crud.job_script.create(**fill_job_script_data())
+
+    job_script_file_name = "entrypoint.sh"
+
+    await synth_services.file.job_script.upsert(
+        parent_id=base_job_script.id,
+        filename=job_script_file_name,
+        upload_content=job_script_data_as_string,
+        file_type="ENTRYPOINT",
+    )
+
+    inserted_job_script_id = base_job_script.id
+    slurm_job_id = 1234
+
+    create_data = fill_job_submission_data(
+        job_script_id=inserted_job_script_id,
+        slurm_job_id=slurm_job_id,
+        status=JobSubmissionStatus.ABORTED,
+        owner_email=tester_email,
+    )
+    original_instance = await synth_services.crud.job_submission.create(**create_data)
+
+    new_owner_email = "new_" + tester_email
+
+    inject_security_header(new_owner_email, permission)
+    response = await client.post(f"/jobbergate/job-submissions/clone/{original_instance.id}")
+
+    assert response.status_code == status.HTTP_201_CREATED, f"Clone failed: {response.text}"
+    response_data = response.json()
+
+    assert response_data["cloned_from_id"] == original_instance.id
+    assert response_data["id"] != original_instance.id
+    assert response_data["owner_email"] == new_owner_email
+    assert response_data["slurm_job_id"] is None
+    assert response_data["status"] == JobSubmissionStatus.CREATED
+
```

Comment:
I think you need to include checks here for the timestamps being updated.

Reply:
Solved by [7883b10](https://github.com/omnivector-solutions/jobbergate/pull/643/commits/7883b10a9462d051e4dd414c781c958d4e9350a5)